### DISCUSSION TITLE
断开连接后无法搜索和重新连接

### DISCUSSION THREAD

**Discussion by leruge** [kevincar/bless#165] [Q&A]
我要在树莓派实现一个蓝牙服务器，方便通过蓝牙连接树莓派，但是只能连接一次，要可以重复被搜索到和连接，应该如何实现？

**Comment by kevincar** [comment]
This seems more like a question than an issue for now; we can move this to a discussion.

As long as the server is running, whatever device you are using to connect to the Raspberry Pi should find it and be able to re-establish a connection. If you're simply testing this out using the example script, the example script closes the server program.
